providers/openstack/scs/cluster-class/templates/cluster-class.yaml

```diff
@@ -38,9 +38,9 @@ spec:
       schema:
         openAPIV3Schema:
           type: string
-          default: "ebfe5546-f09f-4f42-ab54-094e457d42ec"
+          default: ""
```

Comment:
I suggest removing the default and keeping the uuid4 format. You probably need to modify patch `enabledIf` condition